lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -98,6 +98,13 @@ aaa_server_group:
   vrf:
     default_value: "default"
 
+dnsclient:
+  name_server:
+    # NOTE: This does not account for vrf ip name-server
+    config_get: 'show hosts'
+    config_get_token: '/Name servers.* (\S+)$/'
+    config_set: '<state> ip name-server <ip>'
+
```

Comment:
I will let others comment on this as well, but I think we want a default here even it it's set to empty string indicating not configured.

default_value: ''